db/migrations/create_twitter_table.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE tweets(id SERIAL PRIMARY KEY, tweet VARCHAR(240), created_at now());
```

Comment:
The code on the top line doesn't work for me - I think it should be without the end part (created_at_now))

Reply:
yeah I forgot to change that, I actually ran it differently but it forgot the documentation I used and then forgot to look it up and update it lol